app/javascript/components/application_form/preferences/PreferencesSection.js

```diff
@@ -23,7 +23,12 @@ let fieldMapper = {
   Individual_preference: "individualPreference",
   Certificate_Number: "certificateNumber",
   Type_of_proof: "preferenceProof",
-  Preference_Name: "preferenceName"
+  Preference_Name: "preferenceName",
+  State: "state",
+  Zip_Code: "zipCode",
+  City: 'city',
+  Street: 'address',
+  'RecordType.DeveloperName': 'recordTypeDevName'
```

Comment:
We also need a `shortformPreferenceID` here, so that we're not creating a new application preference when fields/settings are updated.

Reply:
@fddayan I'm not seeing `shortformPreferenceID` being submitted to Salesforce, and the application preference records are still being overwritten.

Reply:
Looks like this is more an issue of how LAP handles preferences compared to Salesforce (overwriting with new application preferences, rather than updating the empty ones already present on the application). We can update that in a separate story rather than hold back this PR.